content/theme/races/aqualish.md

```diff
@@ -2,7 +2,7 @@
 title: "Aqualish"
 date: 2020-10-25T23:34:39-07:00
 draft: false
-image: /img/races/aqualish.jpeg
+image: /img/races/aqualish.png
```

Comment:
Does this file exist?

Reply:
I opened a pull request for it I thought.